src/robusta/runner/main.py

```diff
@@ -18,6 +19,11 @@ def main():
     loader = ConfigLoader(registry, event_handler)
     if os.environ.get("ENABLE_MANHOLE", "false").lower() == "true":
         manhole.install(locals=dict(getmembers(robusta_api)))
+
+    sentry_dsn = os.environ.get("SECRET_SENTRY_DSN", "")
+    if sentry_dsn:
+        sentry_sdk.init(sentry_dsn, traces_sample_rate=0.5)
```

Comment:
I would take that value from an env variable, and put the default there.
In case we'll need to change it, we can override using an env variable rather than creating a new code version.

see env_vars.py